### PR TITLE
Updates To JSON Entries

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Revision of my last PR, that crossed streams with another General JSON entry PR. This version merges the last PR with the new General JSON entry

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/json/json.md [hunk 1]
```diff
@@ -7,47 +7,69 @@ Subjects:
...
+  "model" : "Silverado",
+  "miles" : 27500,
+  "year" : 2020,
+  "owner" : {
+              "firstName" : "John",
+              "lastName" : "Doe"
+              },
+  "features" : ["4WD","Towing Package","Lift Kit"],
+  "lease" : false
+}
```
Comment (Dusch4593) [inline content/general/concepts/json/json.md:67]:
```suggestion
{
  "make" : "Chevy",
  "model" : "Silverado",
  "miles" : 27500,
  "year" : 2020,
  "owner" : {
    "firstName" : "John",
    "lastName" : "Doe"
   },
  "features" : ["4WD", "Towing Package", "Lift Kit"],
  "lease" : false
}
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Only one suggestion from me 😄

**Comment by SSwiniarski** [comment]
@Dusch4593, I applied your suggestion.

File: content/general/concepts/json/json.md [hunk 2]
```diff
@@ -7,47 +7,69 @@ Subjects:
...
   - 'JSON'
   - 'Objects'
+  - 'Data Structures'
 CatalogContent:
   - 'introduction-to-javascript'
   - 'paths/full-stack-engineer-career-path'
 ---
 
-JavaScript Object Notation (JSON) is a data format that is readable, writable, and parsable for both humans and machines. JSON is based on the syntax of the third edition of [JavaScript](https://www.codecademy.com/resources/docs/javascript) (ECMA-262) that was released back in December 1999. However, it is language-independent.
+JavaScript Object Notation (JSON) is a data format that is readable, writable, and parsable for both humans and machines. JSON is based on the syntax of the third edition of [JavaScript](https://www.codecademy.com/resources/docs/javascript) (ECMA-262) that was released back in December 1999. However, it is language-independent. Many programming languages, such as [Python](https://www.codecademy.com/resources/docs/python), have implemented [libraries](https://www.codecademy.com/resources/docs/python/json-module) to parse and generate JSON-formatted data.
```
Comment (Dusch4593) [inline content/general/concepts/json/json.md:16]:
```suggestion
**JavaScript Object Notation (JSON)** is a data format that is readable, writable, and parsable for both humans and machines. JSON is based on the syntax of the third edition of [JavaScript](https://www.codecademy.com/resources/docs/javascript) (ECMA-262) that was released back in December 1999. However, it is language-independent. Many programming languages, such as [Python](https://www.codecademy.com/resources/docs/python), have implemented [libraries](https://www.codecademy.com/resources/docs/python/json-module) to parse and generate JSON-formatted data.
```

One last super minor suggestion, per our updated [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/#bold) re: bolded text.

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM! 👍🏻 Just one small suggestion, but I think this is ready for at least one more review.

**Comment by SSwiniarski** [comment]
@Dusch4593, committed last suggestion.

File: content/general/concepts/json/json.md [hunk 3]
```diff
@@ -7,47 +7,69 @@ Subjects:
...
-    "propertyC": "valueC",
-  }
-]
-```
+- Number: A signed decimal number. JSON makes no distinction between integer and floating point.
+- String: A sequence of zero or more Unicode characters enclosed in double quotes.
+- Boolean: Either `true` or `false`.
+- Array: An ordered, comma delimited, list of zero or more elements of one of JSON's data types, enclosed in square brackets.
+- Object: A collection of name-value pairs inside curly brackets.
+- `null`: An empty value represented by the word `null`.
```
Comment (Dusch4593) [inline content/general/concepts/json/json.md:43]:
```suggestion
- [Array](https://www.codecademy.com/resources/docs/general/array): An ordered, comma delimited, list of zero or more elements of one of JSON's data types, enclosed in square brackets.
- [Boolean](https://www.codecademy.com/resources/docs/general/boolean): Either `true` or `false`.
- [Number](https://www.codecademy.com/resources/docs/general/number): A signed decimal number. JSON makes no distinction between integer and floating point.
- [Object](https://www.codecademy.com/resources/docs/general/object): A collection of name-value pairs inside curly brackets.
- [String](https://www.codecademy.com/resources/docs/general/string): A sequence of zero or more Unicode characters enclosed in double quotes.
- `null`: An empty value represented by the word `null`.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Just one suggestion 😄

**Comment by SSwiniarski** [comment]
@Dusch4593, done. Any more suggestions?

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@SSwiniarski looks pretty good, just a minor comment

**Comment by SSwiniarski** [comment]
@caupolicandiaz, I implemented your suggestion.

**Comment by Dusch4593** [comment]
@SSwiniarski I just merged #789 and the [general JSON entry](https://www.codecademy.com/resources/docs/general/json) should be updated. Sorry about crossing the streams with this one (again on my part). Fwiw, I transferred every change you'd made in this PR.

To avoid resolving conflicts, I'm going to delete the `json.md` file from this PR and then we can continue with the other files. 😄

**Comment by SSwiniarski** [comment]
@Dusch4593, I updated the json.md in this branch so it matches the current one, so now it's just a couple of added links. @caupolicandiaz, if yo can approve this, I think we can merge.

File: content/general/concepts/json/json.md [hunk 4]
```diff
@@ -7,33 +7,36 @@ Subjects:
...
+<link rel="canonical" href="https://www.codecademy.com/article/what-is-json" />
+
+**JavaScript Object Notation (JSON)** is a language-independent data format that is readable, writable, and parsable for both humans and machines. JSON is based on the syntax of the third edition of a [JavaScript](https://www.codecademy.com/resources/docs/javascript) standard known as ([ECMAScript](https://www.ecma-international.org/publications-and-standards/standards/ecma-262/). Many programming languages, such as [Python](https://www.codecademy.com/resources/docs/python), have implemented [libraries](https://www.codecademy.com/resources/docs/python/json-module) to parse and generate JSON-formatted data.
+
+JSON is commonly used for transferring data between clients and servers for tasks such as web browsing or form submission. Some companies also use JSON to allow their data to use in other applications via [API](https://www.codecademy.com/resources/docs/general/api). Some examples include:
```
Comment (caupolicandiaz) [inline content/general/concepts/json/json.md:19]:
```suggestion
JSON is commonly used for transferring data between clients and servers for tasks such as web browsing or form submission. Some companies also use JSON to allow their data to be accessed in other applications via [API](https://www.codecademy.com/resources/docs/general/api). Some examples include:
```

**Review by caupolicandiaz — COMMENTED** [review]
@SSwiniarski looks pretty good, just one small edit

**Review by caupolicandiaz — APPROVED** [review]
@Dusch4593 good to go

**Pull request merged**